wiki/content/deploy/ludicrous-mode.md

```diff
@@ -8,33 +8,41 @@ weight = 13
+In this mode, Dgraph doesn't wait for mutations to be applied. When a mutation comes, it proposes the mutation to the cluster and as soon as the proposal reaches the other nodes, it returns the response right away. You don't need to send a commit request for mutations. It's equivalent of having `CommitNow` set automatically for all mutations. All the mutations are then sent to background workers which apply them repeatedly.
 
-Also, Dgraph does not sync writes to disk. This increases throughput but may result in loss of unsynced writes in the event of hardware failure.
+Also, Dgraph does not sync writes to disk. This increases throughput but may result in loss of un-synced writes in the event of hardware failure.
 
 
-## What is the trade off?
+## What is the trade-off?
 
-As mentioned in the section above, it provides amazing speed at the cost of some guarantees.
+As mentioned above, Ludicrous mode provides amazing speed at the cost of some guarantees.
 
-It can be used when we have write-heavy operations and there is a time gap between queries and mutations, or you are fine with potentially reading stale data.
+It can be used to handle write-heavy operations when there is a time gap between queries and mutations, or when you are fine with potentially reading stale data.
 
-There are no transactions in ludicrous mode. That is, you cannot open a transaction, apply a mutation, and then decide to cancel the transaction. Every mutation request is committed to Dgraph.
+There are no transactions in Ludicrous mode. That is, you cannot open a transaction, apply a mutation, and then decide to cancel the transaction. Every mutation request is committed to Dgraph.
 
-## Can the cluster run with HA?
+## Can you use Ludicrous mode in a highly-available (HA) cluster?
 
-Yes, ludicrous mode works with the cluster set up in a highly-available (HA) configuration.
+Yes, Ludicrous mode works with a cluster set up in a highly-available (HA) configuration.
 
-## Can the cluster run with multiple data shards?
+## Can a cluster run with multiple data shards?
 
-Yes, ludicrous mode works with the cluster set up with multiple data shards.
\ No newline at end of file
+Yes, Ludicrous mode works with the cluster set up with multiple data shards.
+
+## How does Ludicrous mode handle concurrency?
+
+Ludicrous mode now runs mutations concurrently per predicate. This is enabled
+by default with 2000 concurrent threads available, but you can adjust the number
+of concurrent threads available using the `--ludicrous_concurrency` configuration setting on the Alpha and Zero nodes in a cluster.
```

Comment:
This config flag only exists in `dgraph alpha`.

Reply:
Fixed! Thank you 👍